controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -296,6 +296,23 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 				return reconcile.Result{}, err
 			}
 
+			//TODO: Verify the caps
+			if err := r.reconcileCephClientNfsProvisioner(
+				consumerResources.GetCsiNfsProvisionerSecretName(),
+				consumerResources.GetSubVolumeGroupName(),
+				consumerConfigMap,
+			); err != nil {
+				return reconcile.Result{}, err
+			}
+
+			if err := r.reconcileCephClientNfsNode(
+				consumerResources.GetCsiNfsNodeSecretName(),
+				consumerResources.GetSubVolumeGroupName(),
+				consumerConfigMap,
+			); err != nil {
+				return reconcile.Result{}, err
+			}
```

Comment:
Shouldn't this be behind a condition?

Reply:
Right, we should put it behind if we have NFS enabled on storageCluster spec